src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -348,57 +345,46 @@ public void clickAjaxNoResponsePanel() {
     public void clickViewPhotoLink(String panelBodyIndex, String urlRegex) {
         String idOfPanelBody = "panelBodyCollapse-" + panelBodyIndex;
         WebElement photoCell = browser.driver.findElement(By.id(idOfPanelBody))
-                                             .findElements(By.cssSelector(".profile-pic-icon-click"))
-                                             .get(0);
+                                             .findElement(By.cssSelector(".profile-pic-icon-click"));
         executeScript("document.getElementById('" + idOfPanelBody + "')"
                       + ".getElementsByClassName('profile-pic-icon-click')[0]"
                       + ".getElementsByTagName('a')[0].click();");
         Actions actions = new Actions(browser.driver);
 
         actions.moveToElement(photoCell).perform();
-        waitForElementPresence(By.cssSelector(".popover-content > img"));
 
-        List<WebElement> photos = browser.driver.findElements(By.cssSelector(".popover-content > img"));
-        AssertHelper.assertContainsRegex(urlRegex, photos.get(photos.size() - 1).getAttribute("src"));
+        AssertHelper.assertContainsRegex(urlRegex,
+                waitForElementPresence(By.cssSelector(".popover-content > img")).getAttribute("src"));
     }
 
     public void hoverClickAndViewStudentPhotoOnHeading(String panelHeadingIndex, String urlRegex) {
         String idOfPanelHeading = "panelHeading-" + panelHeadingIndex;
-        WebElement photoDiv = browser.driver.findElement(By.id(idOfPanelHeading))
-                                            .findElement(By.className("profile-pic-icon-hover"));
-        executeScript("arguments[0].scrollIntoView(true);", photoDiv);
-        Actions actions = new Actions(browser.driver);
-        actions.moveToElement(photoDiv).perform();
-        waitForElementPresence(By.cssSelector(".popover-content"));
 
-        executeScript("document.getElementsByClassName('popover-content')[0]"
-                      + ".getElementsByTagName('a')[0].click();");
+        /*
+         * Execute JavaScript instead of using Selenium selectors to bypass bug
```

Comment:
This was [copied](https://github.com/TEAMMATES/teammates/blob/61839efa09d73ef906ba5985d6927d0d8a8c9b91/src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java#L411) haha, but I think I'll revert this part. Looks like the Selenium selectors do work on Travis.